Mathlib/Analysis/InnerProductSpace/Harmonic/Basic.lean

```diff
@@ -0,0 +1,146 @@
+/-
+Copyright (c) 2025 Stefan Kebekus. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Stefan Kebekus
+-/
+import Mathlib.Analysis.InnerProductSpace.Laplacian
+
+/-!
+# Harmonic Functions
+
+This file defines harmonic functions on real, finite-dimensional, inner product spaces `E`.
+-/
+
+variable
+  {E : Type*} [NormedAddCommGroup E] [InnerProductSpace ℝ E] [FiniteDimensional ℝ E]
+  {F : Type*} [NormedAddCommGroup F] [NormedSpace ℝ F]
+  {G : Type*} [NormedAddCommGroup G] [NormedSpace ℝ G]
+  {f f₁ f₂ : E → F}
+  {x : E} {s t : Set E} {c : ℝ}
+
+open InnerProductSpace Topology
+
```

Comment:
Could you work in the `InnerProductSpace` namespace? Harmonicity is ubiquitous in mathematics, so better not put one in the root namespace.

Reply:
Implemented as suggested.